examples/lusid/corporate-actions-source/create-corporate-action-source.sql

```diff
@@ -0,0 +1,22 @@
+-- ===============================================================
+-- Description:
+-- In this query, we write a corporate action source to LUSID.
+-- ===============================================================
+
+-- Defining corporate action scope and code.
+
+@@corporateActionScope = select 'luminesce-examples';
+@@corporateActionCode = select 'uk-equity';
+
+-- Define a table containing sample information.
+@corporate_action_source_table = 
+
+select  "Sample Actions Source" as DisplayName,
+        @@corporateActionScope as CorporateActionSourceScope,
+        @@corporateActionCode as CorporateActionSourceCode,
+        @@corporateActionScope as InstrumentScope,
```

Comment:
This is a list of instrument scopes and would be input by a comma separated list in string format: "scope1, scope 2, scope3...". This is optional though so I would leave it out.